### PR TITLE
Debug first video

### PR DESCRIPTION
**Pull request by rfayat**

### CODE REVIEW

File: camera.py [hunk 1]
```diff
@@ -31,9 +31,6 @@ def __init__(self, config, path_to_output='videos'):
         self.pipeline = None
         self.queue = None
         self.livedisplay = False
-
-        # logging.basicConfig(filename=path_to_output + '/run.log',
-        #                     level=logging.INFO)
```
Comment (stebett) [inline camera.py:36]: This should be an option, logging to a file for the future may be handy

**Comment by rfayat** [comment]
I switched the order of counter update / comparison with the number of frames in `camera.Camera.new_video`, let me know if it is a good idea.

I also added the logging of the number of frames in the current chunk, got, while expecting chunks of 50 frames:
```bash
$ python -m record -o ~/Desktop/tmp/acquisition --force
WARNING:root:
[!] Video:                     /users/nbc/fayat/Desktop/tmp/acquisition/000000.avi
[!] Number of frames:          98
[!] Expected number of frames: 50
[!] Frames in chunk:           98
[!] Expected in chunk:         50

WARNING:root:
[!] Video:                     /users/nbc/fayat/Desktop/tmp/acquisition/000050.avi
[!] Number of frames:          149
[!] Expected number of frames: 100
[!] Frames in chunk:           99
[!] Expected in chunk:         50

WARNING:root:
[!] Video:                     /users/nbc/fayat/Desktop/tmp/acquisition/000100.avi
[!] Number of frames:          199
[!] Expected number of frames: 150
[!] Frames in chunk:           99
[!] Expected in chunk:         50
```
Looks like the main issue here is that we are getting a frame trigger both for rising and falling edges of the PWM. There might also be an issue of 1 counter sample (maybe it's my fault with the changes I just made).

File: camera.py (hunk 1 above)
Comment (rfayat) [reply camera.py:36]: Oh ok, this bit of code will appear here, we can copy and paste it from github later

**Comment by stebett** [comment]
> I switched the order of counter update / comparison with the number of frames in camera.Camera.new_video, let me know if it is a good idea.

If it works its cool with me

File: camera.py (hunk 1 above)
Comment (stebett) [reply camera.py:36]: Ok

**Pull request merged**